README.md

```diff
@@ -68,7 +68,7 @@ Include the following zendesk package version in your `packages.yml` file:
 ```yml
 packages:
   - package: fivetran/zendesk
-    version: [">=0.21.0", "<0.22.0"]
+    version: 0.22.0-a1
```

Comment:
This will need to be updated

Reply:
Very odd....I added this and pushed everything and `git status` says `nothing to commit, working tree clean` but when I highlight this line in VSCode it says "uncommitted changes" (and it's obviously not here)
![image](https://github.com/user-attachments/assets/d3272b2b-54da-4fcd-90cf-516f2d8efaad)

I'll add a space and see if that does the trick

Reply:
Looks like it worked 🤷